### PR TITLE
Start using the visits human_id

### PR DESCRIPTION
**Pull request by alan**
Use the human_id on emails and the api

### CODE REVIEW

File: app/views/visitor_mailer/booked.html.erb [hunk 1]
```diff
@@ -103,5 +103,5 @@
 </p>
 
 <p>
-  <small><%= t(".visit_id") %> <%= @visit&.id %></small>
+  <small><%= t(".visit_id") %> <%= @visit.human_id %></small>
```
Comment (StupidCodeFactory) [inline app/views/visitor_mailer/booked.html.erb]: I'm wondering wether it is worth keeping the ```&.```?

Comment (alan) [inline app/views/visitor_mailer/booked.html.erb]: I was wondering why it was there in the first place, all visits should have an `id` or `human_id`. If there is a good reason for I can put it back here as well as changing the rest of the emails

Comment (StupidCodeFactory) [inline app/views/visitor_mailer/booked.html.erb]: it's for the email preview i think as we rebuild a new visit object

Comment (StupidCodeFactory) [inline app/views/visitor_mailer/booked.html.erb]: @alan ^

Comment (alan) [inline app/views/visitor_mailer/booked.html.erb]: I've just tried and it works

**Pull request merged**